### PR TITLE
AEA-2094: Pact tests for dispense notification amend

### PR DESCRIPTION
**Pull request by callumtipper-IW**

### CODE REVIEW

File: tests/e2e/pact/resources/common.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ export const basePath = "/FHIR/R4"
 
 export type ApiMode = "live" | "sandbox"
 export type ApiEndpoint = "prepare" | "process" | "task" | "validate" | "verify-signature" | "metadata" | "tracker"
-export type ApiOperation = "send" | "cancel" | "dispense" | "release" | "return" | "withdraw" | "claim"
+export type ApiOperation = "send" | "cancel" | "dispense" | "dispenseamend" |
```
Comment (JackRobinsonIW) [inline tests/e2e/pact/resources/common.ts:8]: does this need to be a single string `dispenseamend` could we do `dispense-amend` instead?

File: tests/e2e/pact/services/update-prescriptions.ts [hunk 2]
```diff
@@ -29,10 +30,13 @@ const x509CertificatePath = process.env.SIGNING_CERT_PATH
 
 const isProd = process.env.APIGEE_ENVIRONMENT === "prod"
 
+let amendBundleIdentifier = null
```
Comment (JackRobinsonIW) [inline tests/e2e/pact/services/update-prescriptions.ts:33]: don't think we need this to be a top level variable, could sit inside the `updatePrescriptions` function I tihnk

File: tests/e2e/pact/resources/common.ts (hunk 1 above)
Comment (callumtipper-IW) [reply tests/e2e/pact/resources/common.ts:8]: Couldn't do this for the string matching reasons discussed

**Review by JackRobinsonIW — APPROVED** [review]

**Pull request merged**